demo/.envrc

```diff
@@ -0,0 +1 @@
+use nix
```

Comment:
Why adding this?

Reply:
For this shortcut / convenience: https://github.com/input-output-hk/hydra-poc/blob/810a9c2cfe7c4b09538a6e8d34a1d52a8cda45c2/docs/docs/getting-started/demo/without-docker.md?plain=1#L29